### PR TITLE
cloud: update cloud node status

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update cloud node status

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by SunRunAway — APPROVED** [review]

File: tidb-cloud/monitor-tidb-cluster.md [hunk 1]
```diff
@@ -54,6 +61,10 @@ In TiDB Cloud, you can view the commonly used metrics of a cluster from the foll
 
 The cluster overview page provides general metrics of a cluster, including total QPS, query duration, active connections, TiDB CPU, TiKV CPU, TiFlash CPU, TiDB memory, TiKV memory, TiFlash memory, TiKV used storage size, and TiFlash used storage size.
 
+> **Note:**
+>
+> Some of these metrics might be available only in Dedicated Tier clusters.
```
Comment (TomShawn) [inline tidb-cloud/monitor-tidb-cluster.md:66]:
```suggestion
> Some of these metrics might be available only for Dedicated Tier clusters.
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 2]
```diff
@@ -26,22 +26,29 @@ You can see the current status of each running cluster on the cluster page.
...
 
+> **Note:**
+>
+> The TiDB node status is only available for Dedicated Tier clusters.
+
 | TiDB node status | Description |
 |:--|:--|
-| **Normal** | Normal running |
-| **Creating** | Creating the node |
+| **Available** | The TiDB node is healthy and available |
```
Comment (ran-huang) [inline tidb-cloud/monitor-tidb-cluster.md:35]: To be consistent with Cluster Status
```suggestion
| **AVAILABLE** | The TiDB node is healthy and available. |
```

File: tidb-cloud/monitor-tidb-cluster.md [hunk 3]
```diff
@@ -26,22 +26,29 @@ You can see the current status of each running cluster on the cluster page.
 
 ### TiDB node status
```
Comment (Oreoxmt) [inline tidb-cloud/monitor-tidb-cluster.md:27]: Please modify L14 `### Cluster Status` to `### Cluster status`.

File: tidb-cloud/monitor-tidb-cluster.md (hunk 2 above)
Comment (Oreoxmt) [reply tidb-cloud/monitor-tidb-cluster.md:35]: > To be consistent with Cluster Status

image

File: tidb-cloud/monitor-tidb-cluster.md (hunk 3 above)
Comment (TomShawn) [reply tidb-cloud/monitor-tidb-cluster.md:27]: done

**Review by ran-huang — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**